src/bin/psql/model.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MODEL_H_4D955674
+#define MODEL_H_4D955674
+
+#include "rowmap.h"
+
+typedef struct Model
+{
+	RowMap *rowmap;
+	size_t* maxlens;
+	size_t nfields;
+
+	Row header;
+
+} Model;
+
+#define MIN(a, b)  (((a) < (b)) ? (a) : (b))
```

Comment:
These are already available as `Min` and `Max`, which are defined in `c.h`, which is imported via `postgres_fe` in `psql`.